storage/src/adb/current.rs

```diff
@@ -177,19 +173,23 @@ impl<
             for _ in pruned_bits..mmr_pruned_leaves {
                 status.append(false);
             }
+
+            grafter
```

Comment:
For consistency, may make sense to put a comment here.

Reply:
done